validation.md

```diff
@@ -2241,6 +2244,84 @@ Occasionally, you may want to attach additional validation rules to your default
         // ...
     });
 
+<a name="validating-emails"></a>
+## Validating Emails
+
+Laravel provides a variety of validation rules that may be used to validate uploaded files, such as `rfc`, `strict` and `dns`. While you are free to specify these rules individually when validating emails, Laravel also offers a fluent file validation rule builder that you may find convenient:
```

Comment:
You probably mean "email" instead of "file":
```suggestion
Laravel provides a variety of validation rules that may be used to validate uploaded files, such as `rfc`, `strict` and `dns`. While you are free to specify these rules individually when validating emails, Laravel also offers a fluent email validation rule builder that you may find convenient:
```